### ISSUE TITLE
global definition

### ISSUE THREAD

**Issue by FedericoTartarini** [CenterForTheBuiltEnvironment/pythermalcomfort#47]
This variable is defined inside an if statement and the code may break if the conditions are not met since hc_natural may not be defined. Please either change the elif to an else or move the assignment of this variable outside the if statement.

https://github.com/CenterForTheBuiltEnvironment/pythermalcomfort/blob/c847e76471ad6c02548bc2d6458b6b50b51dd098/src/pythermalcomfort/jos3_functions/thermoregulation.py#L75

**Comment by FedericoTartarini** [comment]
The same problem is true for `hr` in line 307 of the same file (`thermoregulation.py`)

**Comment by AkihisaNomoto** [comment]
This function `conv_coef` has a defalt posture "standing" so I don't expect the code breaks unless the user mistyped posture something wrong like "stending". However, I added else statement raising ValueError if that happens. I wrote the same else statement in `rad_coef` function in `thermoregulation.py`.

I also changed this function (`conv_coef`) a bit - I wrote a function called `natural_convection` and `forced_convection` inside this `conv_coef` function for the users to easily to understand.
